app/src/main/java/pl/droidevs/books/login/LoginActivity.java

```diff
@@ -43,20 +44,20 @@ protected void onCreate(Bundle savedInstanceState) {
         AndroidInjection.inject(this);
         this.unbinder = ButterKnife.bind(this);
 
-        textHello.setText(loginService.getUserName());
+        //textHello.setText(loginService.getUserName());
```

Comment:
Nie zostawiamy zakomentowanego kodu - odkłada on się niepotrzebnie w repozytiorum. Ja jak widzę zakomentowany kod automatycznie go usuwam, nie zastanawiam się ani nie analizuję nawet co robi 😄

Reply:
Wybacz. Starałem się usunąć wszystkie komentarze przed commitem ale to przeoczyłem.